modules/core/tests/server/userModel.test.server.js

```diff
@@ -86,6 +89,32 @@ describe('User functionality', function () {
 
   });
 
+  it('should not update the password, when updating the user', function (done) {
+    User.update({ passwordResetToken: '1234567890' }, { where : { email: 'test@test.com' }})
+      .then(function () {
+        return User.findOne({ where : { email: 'test@test.com' }});
+      })
+      .then(function (user) {
+        expect(user.password).toEqual(savedUser.password);
+        done();
+      })
+      .catch(done.fail);
+
+  });
+
+  it('should hash a new password, when updating the user', function (done) {
```

Comment:
it should hash a new password when updating the password field of the User model